packages/next-sample/libraries/logto.ts

```diff
@@ -0,0 +1,13 @@
+import NodeClient from '@logto/next';
+
+export const logtoClient = new NodeClient({
+  appId: 'foo',
+  endpoint: 'https://logto.dev',
+  baseUrl: 'http://localhost:3000',
+  signInEndpoint: '/api/sign-in',
+  signInCallbackEndpoint: '/api/sign-in-callback',
+  signOutEndpoint: '/api/sign-out',
+  userEndpoint: '/api/user',
```

Comment:
i think this is too much configs. maybe only `baseUrl` is required while other endpoints has default values? e.g.
```ts
{
  baseUrl: 'http://localhost:3000/api',
  // other endpoints are optional
}
```

Reply:
or we can add these options until we need them?

Reply:
removed.